### DISCUSSION TITLE
Should you disable each utilities individually that you don't use in tailwind?

### DISCUSSION THREAD

**Discussion by rifqirosyidi** [tailwindlabs/tailwindcss#3727] [Help] [answered]
Hi, there

I developed a website using tailwindcss and I'm wondering about the section of disabling utilites. If I'm not using some of the utilities, should I set some utilities to false in `corePlugins` section of my config file. or just simply keep it blank, when there are so many utilities that have to be disabled?

**Comment by NovusOpus** [comment, marked as answer]
If you have it configured with Purge CSS, you don't really have to do that, Since All unused classes or utilities will be purged in your Final ready for production build. More info about it here: https://tailwindcss.com/docs/optimizing-for-production

I think that's for some very specific case scenarios, But hope you get a more clear answer from TailwindCSS staff/team.
